internal/processors/signalconvert/signalconvert.go

```diff
@@ -197,6 +201,10 @@ func signalEqual(a, b vss.Signal) bool {
 	return a.Name == b.Name && a.Timestamp.Equal(b.Timestamp) && a.TokenID == b.TokenID
 }
 
+func signalMaxValue(a vss.Signal) bool {
+	return a.ValueNumber == 255 || a.ValueNumber == 65535
```

Comment:
Consider using the [`math` package constants](https://pkg.go.dev/math#pkg-constants) `MaxUint8` and `MaxUint16`.